src/main/java/ja/workshops/spring/części/konfiguracja/xml/c_mvc/package-info.java

```diff
@@ -0,0 +1,19 @@
+/**
+ *
+ * <h1>Prezentacja konfiguracji XML-owej!</h1>
+ *
+ *  <p>Ostatnim zadaniem będzie konfiguracja prostej springowej aplikacji MVC.</p>
+ *  <p>W tym pakiecie znajdziesz podstawowe komponenty obrazujące podział aplkiacji na model, widok i kontroler.</p>
```

Comment:
Aplkiacji? Apli - ka - cja, nie aplki - a - cja ;-)
Słowo o MWK poproszę, skoro ćwiczenie wymaga znajomości. Możesz odesłać do mnie, albo zrobić dwa zdania opisu, że MWK to wzorzec podziału odpowiedzialności z rozmaitymi implementacjami i wariantami.